packages/core/admin/admin/src/content-manager/components/AttributeFilter/Filters.js

```diff
@@ -21,7 +21,7 @@ const Filters = ({ displayedFilters }) => {
 
   return (
     <>
-      <Box paddingTop={1} paddingBottom={1}>
```

Comment:
Are you sure this doesn't break the alignment? If not: can we remove the `Box` all together because it doesn't do anything anymore?

Reply:
Yes, it doesn't break the alignment, even with other menus like the i18n fil
<img width="1068" alt="alignment" src="https://github.com/strapi/strapi/assets/2589748/19885d4f-b3ea-4543-bd72-056952882758">
<img width="1041" alt="align-i18n" src="https://github.com/strapi/strapi/assets/2589748/849f59c9-cda7-40d9-8323-1bfccfa61918">
ter as you can see in these two pics